.github/workflows/ansible-test.yml

```diff
@@ -7,15 +7,15 @@ on:
     # * is a special character in YAML so you have to quote this string
     - cron: '0 8 * * *'
 env:
-  python_version: '3.10'
+  python_version: '3.11'
```

Comment:
is python 3.11 required?

Reply:
Ansible 2.18 now requires a minimum of Python 3.11